### ISSUE TITLE
Is it possible to unload a resource?

### ISSUE THREAD

**Issue by slavafomin** [ocombe/ocLazyLoad#97]
Hello! Thanks for this module!

I have a question.

Is it possible to unload a specific resource?

I have two parts of my application that are using two different conflicting stylesheets. I want to use `ocLazyLoad` to load CSS dynamically by the router for each of these two routes. I will need a way to remove loaded stylesheet when navigating from one route to the other.

Please advise. Thanks!

**Comment by curlyfro** [comment]
you can't remove resources after they've been loaded.  jquery and angular can't fix this problem.  browser manufacturers would have solved this a long time ago if it were that simple.

**Comment by kloy** [comment]
@curlyfro that is true for JavaScript not CSS. Stylesheets can be disabled via disable attribute or removed and the page will no longer utilize those styles.

My recommendation on this is to not manage stylesheets directly in ocLazyLoad but use a ng-switch at the top of your body tag.

**Comment by edu-nixon** [comment]
anything new on this?
